src/pages/request_demo.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Layout from '../components/Layout'
+import { StartupForm } from './startups'
+import { Col, Row } from 'antd'
+
+function submitForm(e) {
+  e.preventDefault()
+  var data = {}
+  data['form'] = 'Demo Request'
+  data['firstName'] = document.getElementById('firstName').value
+  data['lastName'] = document.getElementById('lastName').value
+  data['email'] = document.getElementById('email').value
+  data['company'] = document.getElementById('company').value
+
+  var xhr = new XMLHttpRequest()
+  xhr.open(
+    'POST',
+    'https://8qyzxcmhxf.execute-api.us-east-1.amazonaws.com/Prod/submitForm',
+    true
+  )
+  xhr.setRequestHeader('Accept', 'application/json; charset=utf-8')
+  xhr.setRequestHeader('Content-Type', 'application/json; charset=UTF-8')
+  xhr.send(JSON.stringify(data))
+
+  xhr.onloadend = response => {
+    if (response.target.status === 200) {
+      window.location.href = '/application-received'
```

Comment:
This redirect really confused me. Just a super simple extra pages that's just "We'll be in touch to schedule a demo. In the mean time, you can try PostHog out for free"

Reply:
Will do